### DISCUSSION TITLE
[Feature request/Examples required]: Animated Scatter graphs

### DISCUSSION THREAD

**Discussion by MacaScull** [airbnb/visx#1400] [Q&A]
Hello, Im pretty new to using visx but I was wondering if you already have this feature or an example on how to build it.

I want to build an animated scatter plot with circles over time, like the line radial plot: https://airbnb.io/visx/lineradial or like this d3 example: https://medium.com/@louisemoxy/create-a-d3-line-chart-animation-336f1cb7dd61 https://codepen.io/louisemoxy/pen/qMvmBM

I also would like commands to pause/fast forward/reverse if that possible.

**Comment by williaster** [comment]
Hey @MacaScull, visx isn't really opinionated on animation (since there are many JS/TS animation libraries out there) but [`react-spring`](https://react-spring.io/) is our favorite for animating things in a performant way in react (animation in react has more perf challenges than straight JS)

animating circles is pretty different from animating a line chart. for the latter, I think the lineradial is the best example we have. for the former, we don't have a good standalone example though we do animate circles/glyphs in the `@visx/xychart` package – it's source code is actually pretty well isolated [link](https://github.com/airbnb/visx/blob/master/packages/visx-xychart/src/components/series/private/AnimatedGlyphs.tsx). Hope that helps, if you (or someone else) makes a better standalone demo feel free to post back here!
